### ISSUE TITLE
[SUGGESTION] WZS: Cheaper Gas Mask variant

### ISSUE THREAD

**Issue by Azurblau** [Azurblau/Zombies.Zone#153]
One that dims but not entirely disables the nausea effect caused by sphynx and only halves poison damage. Or perhaps some shades just for the sphynx effect?

**Comment by Loickun** [comment]
No, that would make sphinx completly useless in no time. The health it regens doesnt outheal the damage the humans deal

**Comment by jems01** [comment]
"chinese gas mask variant"

**Comment by mikan1216** [comment]
Maybe some kind of "air filtering mask"? Makes you resistant to poison, but for certain amount of damage added up, but doesn't protect from nausea effect
